lib/core/db_models/worker/locationModel.dart

```diff
@@ -17,16 +17,16 @@ class LocationModelDetail {
           ? json[LocationSchema.formattedAddress]
           : "",
       coordinates: json[LocationSchema.coords] != null
-          ? json[LocationSchema.coords]
+          ? Coordinates.fromJson(json[LocationSchema.coords])
           : null,
       address: json[LocationSchema.address] != null
```

Comment:
Is this model being used anywhere in the app? and I it does did you check it at all places?

Reply:
i have created that model,
this is new,
not being used anywhere rather than the code i have implemented